draft-dekater-scion-dataplane.md

```diff
@@ -1342,20 +1355,97 @@ This section describes the steps that a SCION egress border router must perform
+
+Currently, SCION as described in this document does not provide this type of protection. Endpoints may optionally uses the SCION Packet Authentication Option (SPAO) extension that provides integrity protection for the path header, allowing detection of such attack. This extension is described in [CHUAT22].
+
+Moreover, packet integrity protection is not enough if there are two colluding adversaries on the path. These colluding adversaries can forward the packet between them using a different path than selected by the source endpoint: The first on-path attacker remodels the packet header arbitrarily, and the second on-path attacker changes the path back to the original source-selected path, such that the integrity check by the destination endpoint succeeds. To prevent this attack and to defend against multiple on-path adversaries in general, path validation is required (which is currently not supported by SCION).
+
+
+#### Modification of the Address Header
+
+Besides modifying the SCION path header, an on-path adversary can also modify the source and destination addresses in the SCION address header. If an adversary changes the address of the destination endpoint, it needs to modify the "future" path between itself and the new destination accordingly; otherwise the packet would simply be dropped by the last AS on the path.
+
+Modifications of the SCION address header can be discovered by the destination endpoint if the packet is integrity protected (which is currently not supported by SCION).
+
+
+### Off-Path Attacks - Volumetric Denial of Service
+
+SCION's path-awareness limits the abilities of an off-path adversary to influence forwarding in the data plane. An adversary can merely attempt to disrupt the connectivity of the chosen path choosen by an endpoint, and force the source endpoint to select a new path. One way to do this is by flooding a link on the path with excessive traffic; that is, performing a volumetric denial of service attack. Should this happen, the endpoint cna switch to another, non-congested path thanks to multipath, as long as any alternative inter-domain path is available.
+
+SCION also provides protection against so-called reflection-based attacks. Here, the adversary sends high amounts of request packets to a server, but forges the source address of the packets with the address of the victim, triggering the server to send a reply to the victim. In SCION, however, packets contain the path in their packer header, and response packets are simply returned along this contained path to the actual sender. SCION thus prevents from this kind of DoS attacks.
```

Comment:
Is this also valid for vanilla SCION? Could the same attack happen if the attacker is in the same AS as the victim? 
(So, in this case, the attack surface would be greatly reduced, nut it would only be eliminated with SPAO)?